.changes/1508-refactor-attachments.md

```diff
@@ -0,0 +1,2 @@
+- [fix]: Redefined attachments .i.e. due to architectural constraints, they exists just like comments/likes and can be interacted on created pins.
+- [fix]: The url field in pins create/edit is now optional.
```

Comment:
We already discussed on top about the issue of having attachments prior to pin creation. This is a constraint we have because of ```AttachmentManager``` object that lives inside higher class Pins/Events etc. For now, we agreed to have attachments be meant for something that can be interacted after pins are created.